pkg/controller/knativeserving/common/extensions.go

```diff
@@ -56,8 +57,17 @@ func (exts Extensions) Transform(instance *servingv1alpha1.KnativeServing) []mf.
 	for _, extension := range exts {
 		result = append(result, extension.Transformers...)
 	}
-	// Let any config in instance override everything else
 	return append(result, func(u *unstructured.Unstructured) error {
```

Comment:
If we start adding more conditions in here, it might make more sense to define `mf.Transformer` functions elsewhere in the file and reference them here, similar to the `mf.Inject*` functions above (though they wouldn't have to be closures). I'll let you decide whether we do that now or later.

Reply:
Right, if `manifestival` could supply something like `InjectImageURL`, that's will be great.